chopin/views.py

```diff
@@ -72,20 +73,31 @@ class CalendarViewSet(viewsets.ModelViewSet):
 
     def list(self, request, *args, **kwargs):
         try:
-            nb = int(self.request.query_params.get('nb'))
+            start_days = int(self.request.query_params.get('startDays'))
+            end_days = int(self.request.query_params.get('endDays'))
         except TypeError:
-            nb = 7
-        startdate = date.today()
-        enddate = startdate + timedelta(days=nb)
+            start_days = 0
+            end_days = 7
+        today = date.today()
+        start_date = today + timedelta(days=start_days)
+        end_date = today + timedelta(days=end_days)
         queryset = perms_models.Creneau.objects.select_related('perm').filter(
-            perm__semestre=get_current_semester()).filter(date__range=[startdate, enddate])
+            perm__semestre=get_current_semester()).filter(date__range=[start_date, end_date])
         serializer = PermToCalendar(queryset, many=True)
         queryset = chopin_models.Calendar.objects.all().filter(semestre=get_current_semester()).filter(
-            date__range=[startdate, enddate])
+            date__range=[start_date, end_date])
         value = serializer.data
         serializer = CalendarSerializer(queryset, many=True)
         value += serializer.data
-        return Response(value)
+        periodeSortedList= sorted(value, key=lambda d: 3 if d['periode']=="S" else 2 if d['periode']=='D' else 1)
+        sortedList=sorted(periodeSortedList, key=lambda d:d['date'])
+        SortedCalendar={}
```

Comment:
Faudrait du snake case
```suggestion
        sorted_calendar={}
```